cpp/src/platform/windows/LogImpl.cpp

```diff
@@ -41,7 +41,7 @@ errno_t fopen_s(FILE** pFile, const char *filename, const char *mode)
 {
     if (!pFile)
     {
-        _set_errno(EINVAL);
```

Comment:
I'd prefer to use something like this http://lists.fossil-scm.org:8080/pipermail/fossil-users/2013-January/011654.html if you can figure out the correct defines for MINGW